server/src/models/financeTransaction.model.js

```diff
@@ -10,7 +11,8 @@ const FinanceTransactionSchema = mongoose.Schema({
     originatorId: mongoose.ObjectId,
     paymentMethod: String,
     status: { type: String, default: '' }, // calculated on the fly, during fetch
-    deleted: { type: Object, default: null }
+    deleted: { type: DeletedTransactionSchema, default: null },
+    by: String
```

Comment:
Wouldn't we want to keep track of who created and who deleted a finance record separately? That might be two different people.